### PR TITLE
Make clearer where default search operates.

### PR DESCRIPTION
**Pull request by WhiteNoiseDev**
State where default Search operates and which operator affect it behavior

### CODE REVIEW

**Comment by ericaxu** [comment]
Thanks for the PR! Could be a little wordy, can you take a look @marcusolsson ?

File: en/Plugins/Search.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ By default, you can find Search in the left sidebar (magnifying glass icon). You
 
 ## Search terms
 
-A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults.
+A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. By default, the search is performed only over all markdown notes and canvases (both their filenames and their content).
```
Comment (marcusolsson) [inline en/Plugins/Search.md:10]:
```suggestion
A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. By default, Obsidian only searches notes and canvases, including their filenames.
```

According to the [style guide](https://help.obsidian.md/Contributing+to+Obsidian/Style+guide#Notes%2C+files%2C+and+folders), "note" implies Markdown.

**Review by marcusolsson — COMMENTED** [review]
Reading this, it looks like you can't search within, for examples, CSV files, unless you use `file:` or `path:` to include it? Am I correct?

**Comment by WhiteNoiseDev** [comment]
> Reading this, it looks like you can't search within, for example, CSV files, unless you use `file:` or `path:` to include it? Am I correct?

You are wrong.

You can only search within notes and canvases. You can NEVER search within any other file type. 
`File:` and `path:` limit the search to filename and file path, respectively, of all files (beyond notes and canvases) but NOT their content.

**Comment by marcusolsson** [comment]
> You can NEVER search **within** any other file type

The "By default" in the following sentence seems to suggest that you can? If you can't search other files, then I think we should remove it.

> By default, the search is performed only over all markdown notes and canvases

**Comment by WhiteNoiseDev** [comment]
You can search other file types, their name not their content.

**Comment by WhiteNoiseDev** [comment]
The "by default part" can be replaced with "without search operators"

**Comment by ericaxu** [comment]
> State where default Search operates and which operator affect it behavior

I feel like the goal of this PR seems to be to address some kind of concern. I wonder if it would be easier if you file that as an issue instead and let Marcus figure out a way to fix that? Would you prefer that @marcusolsson ?

**Comment by marcusolsson** [comment]
I think I understand know:

- You **cannot** search the contents of files other than notes and canvases.
- By default, you **cannot** search the filenames of files other than notes and canvases.
- If you use the `path` and `file` operators, you **can** search the filenames of files other than notes and canvases.

I can see how this would be confusing, so I think we should document it. 

How about this then?:

1. Update the first text to:

   > Obsidian only searches the contents of notes and canvases.

2. Add a callout that explains this explicitly:
   
   ```plain
   > [!tip] Searching paths and filenames
   > By default, you can only search the paths and filenames of notes and canvases. To search for a path or filename of any file in the vault, use the `path` or `file` operator.
   ```

2. I'm on the fence whether the text in the reference table is needed. If you still think it is with the suggestions above, I'd at least refrain from using passive voice:

   > Find text in filename. Matches any file in the vault.

**Comment by WhiteNoiseDev** [comment]
ok, I would keep all 3 points.

File: en/Plugins/Search.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ By default, you can find Search in the left sidebar (magnifying glass icon). You
 
 ## Search terms
 
-A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults.
+A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. By default, Obsidian only searches notes and canvases, including their filenames.
```
Comment (marcusolsson) [inline en/Plugins/Search.md:10]:
```suggestion
A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. Obsidian only searches the contents of notes and canvases.

> [!tip] Searching paths and filenames
> By default, you can only search the paths and filenames of notes and canvases. To search for a path or filename of any file in the vault, use the `path` or `file` operator.
```

File: en/Plugins/Search.md [hunk 3]
```diff
@@ -39,8 +39,8 @@ Some operators even allow you to add a nested search term within parentheses, fo
...
 |-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `file:`         | Find text in filename.<p/>Example: `file:.jpg` or `file:202209`.                                                                                                                                                     |
-| `path:`         | Find text in file path.<p/>Example: `path:"Daily notes/2022-07"`.                                                                                                                                                    |
+| `file:`         | Find text in filename. All files in the vault are considered.<p/>Example: `file:.jpg` or `file:202209`.                                                                                                                                                     |
```
Comment (marcusolsson) [inline en/Plugins/Search.md:42]:
```suggestion
| `file:`         | Find text in filename. Matches any file in the vault.<p/>Example: `file:.jpg` or `file:202209`.                                                                                                                                                     |
```

File: en/Plugins/Search.md [hunk 4]
```diff
@@ -39,8 +39,8 @@ Some operators even allow you to add a nested search term within parentheses, fo
...
+| `path:`         | Find text in file path. All files in the vault are considered.<p/>Example: `path:"Daily notes/2022-07"`.                                                                                                                                                    |
```
Comment (marcusolsson) [inline en/Plugins/Search.md:43]:
```suggestion
| `path:`         | Find text in file path. Matches any file in the vault.<p/>Example: `path:"Daily notes/2022-07"`.                                                                                                                                                    |
```

**Review by marcusolsson — CHANGES_REQUESTED** [review]

**Pull request merged**